pagerduty/user.go

```diff
@@ -197,10 +201,18 @@ func (s *UserService) ListAll(o *ListUsersOptions) ([]*FullUser, error) {
 func (s *UserService) Create(user *User) (*User, *Response, error) {
 	u := "/users"
 	v := new(UserPayload)
-
 	resp, err := s.client.newRequestDo("POST", u, nil, &UserPayload{User: user}, &v)
 	if err != nil {
-		return nil, nil, err
+		if e, ok := err.(*Error); !ok || strings.Compare(fmt.Sprintf("%v", e.Errors), "[Email has already been taken]") != 0 {
+			return nil, nil, err
+		}
+
+		sUser, sResp, sErr := s.searchExistingUser(user)
```

Comment:
```suggestion
		sUser, sResp, sErr := s.findExistingUser(user)
```

Reply:
Might even be better to findExisting depending on the names of existing functions.

Reply:
Updated